static/app/views/replays/details.tsx

```diff
@@ -82,9 +83,16 @@ function ReplayDetails() {
     );
   }
 
+  // Find LCP spans that have a valid replay node id, this will be used to
+  const highlights: HighlightsByTime = new Map(
+    mergedReplayEvent?.entries[0].data
+      .filter(({op, data}) => op === 'largest-contentful-paint' && data?.nodeId > 0)
+      .map(({start_timestamp, data: {nodeId}}) => [Math.floor(start_timestamp), {nodeId}])
+  );
```

Comment:
This might make sense to live inside the hook or somewhere, for future expansion and tests

Reply:
@ryan953 Applied CR changes to a follow-up I've been working on with canvas: https://github.com/getsentry/sentry/pull/34116